BDT_training/step1_Samplepreparation/runCondor.py

```diff
@@ -35,11 +36,19 @@ def prepare_shell(shell_file, command, condor, FarmDir):
     if args.era == 'all' or args.era == Era:
       Eras.append(Era)
 
-  cmsswBase = os.environ['CMSSW_BASE']
+  cwd = os.getcwd()
+  dir_list = cwd.split('/')
+  cmssw_list = []
+  for dir_ in dir_list:
+    cmssw_list.append(dir_)
+    if 'CMSSW' in dir_: break
+  cmsswBase = '/'.join(cmssw_list)
```

Comment:
Why do you need this? 

old cmsswBase = os.environ['CMSSW_BASE'] should give same results?

Reply:
New DNN application used ROOT v6.28, which is not provided by any CMSSW now. Need new way to obtain this.